<!-- benthos_release.launch -->

<launch>

  <!-- Launch the release script -->
  <node pkg="your_package_name" type="release_script.py" name="benthos_release_node">
  </node>

  <!-- Launch the GUI script -->
  <node pkg="your_package_name" type="gui_script.py" name="benthos_gui_node">
  </node>

</launch>
